tests/memory_checker/test_memory_checker.py

```diff
@@ -225,25 +412,109 @@ def test_memory_checker(duthosts, enum_dut_feature_container, creds, enum_rand_o
     # and number of vm_workers is hard coded. We will extend this testing on all containers after
     # the feature 'memory_checker' is fully implemented.
     container_name = "telemetry"
-    vm_workers = 4
+    vm_workers = 6
```

Comment:
>vm_workers [](http://example.com/codeflow?start=4&length=10)

What does vm_worker mean?
Why magic number 6?

Reply:
`vm_worker` means how many threads `stress` utility will create to consume memory by doing `malloc()/free()` operation. 

Usually each thread will consume the memory around `0MB ~ 256MB`. In order to guarantee total memory usage of all workers is larger than 400MB, I selected the number `6` instead of `4`.